src/main/java/org/broadinstitute/hellbender/utils/codecs/table/TableFeature.java

```diff
@@ -1,26 +1,27 @@
 package org.broadinstitute.hellbender.utils.codecs.table;
 
 
+import htsjdk.samtools.util.Locatable;
 import htsjdk.tribble.Feature;
-import org.broadinstitute.hellbender.utils.GenomeLoc;
-import org.broadinstitute.hellbender.utils.Utils;
 
 import java.util.List;
 
+import static org.broadinstitute.hellbender.utils.Utils.join;
```

Comment:
I don't necessarily agree.  There are definitely times when static imports would confuse things, but I don't think there is much clarity gained by writing Utils all over the place.